README.rst

```diff
@@ -11,10 +42,38 @@ under the hood.
 Installation
 ------------
 
-To use this library, you'll need a Google Cloud Platform project with the Cloud
-Spanner API enabled. For details on enabling the API and authenticating with
-GCP, see the `Cloud Spanner Python client library quickstart guide
-<https://github.com/googleapis/python-spanner/#quick-start>`__.
+Install this library in a `virtualenv`_ using pip. `virtualenv`_ is a tool to
+create isolated Python and Django environments. The basic problem it addresses is one of
+dependencies and versions, and indirectly permissions.
+
+With `virtualenv`_, it's possible to install this library without needing system
+install permissions, and without clashing with the installed system
+dependencies.
+
+.. _`virtualenv`: https://virtualenv.pypa.io/en/latest/
+
+
+Mac/Linux
+~~~~~~~~~
+
+.. code-block:: console
+
+    pip install virtualenv
+    virtualenv <your-env>
+    source <your-env>/bin/activate
+    <your-env>/bin/pip install google-cloud-spanner
```

Comment:
we would also want to install this django library here.

Reply:
Should I for now add a line to simply import the 2.2.24 version?